### DISCUSSION TITLE
4.0.0 alpha 085b094f18a3

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#59] [Announcements]
Blender 4.0.0 Alpha, commit [085b094f18a3](https://projects.blender.org/blender/blender/commit/085b094f18a3).
Build environment:

    Visual Studio 2019, version 16.11.29
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release (78315faf8fbd).

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.0.0-alpha.1-085b094f18a3'>4.0.0 alpha 085b094f18a3</a>.</em>

**Comment by dancingshoes** [comment]
Very happy that Blender 4.0 works!

**Reply by newillusion** [reply]
Alpha-versions are imortant, it contains many delevoper's tools, like "volume sdf" node, shader node preview, new point cloud type, new curve tools and so on... Most of them will be disappear in beta and release versions.
